eng/Subsets.props

```diff
@@ -63,7 +63,8 @@
     <DefaultCoreClrSubsets Condition="'$(PrimaryRuntimeFlavor)' != 'CoreCLR'">clr.iltools+clr.packages</DefaultCoreClrSubsets>
 
     <DefaultNativeAotSubsets>clr.alljits+clr.tools+clr.nativeaotlibs+clr.nativeaotruntime</DefaultNativeAotSubsets>
-    <DefaultNativeAotSubsets Condition="'$(TargetArchitecture)' == 'wasm'">clr.nativeaotlibs+clr.nativeaotruntime+nativeaot.build+mono.wasmruntime</DefaultNativeAotSubsets>
+    <DefaultNativeAotSubsets Condition="'$(TargetsWasm)' == 'true'">clr.nativeaotlibs+clr.nativeaotruntime+nativeaot.build</DefaultNativeAotSubsets>
+    <DefaultNativeAotSubsets Condition="'$(TargetsBrowser)' == 'true'">$(DefaultNativeAotSubsets)+mono.wasmruntime</DefaultNativeAotSubsets>
```

Comment:
This is actually necessary for build correctness - `browser.proj` copies Browser-targeting ICU files to the `libs.native` output directory (duplicatively with `build-native.proj`), and they are not suitable for WASI linking.

Reply:
Should line 66 be TARGETS_WASI?

Reply:
I don't think it should, the subsets it sets are common to both WASI and Browser.